packages/terra-dropdown-button/src/_DropdownList.jsx

```diff
@@ -53,7 +53,7 @@ class DropdownList extends React.Component {
   handleKeyDown(event) {
     const { keyCode } = event;
     const { focused } = this.state;
-    const index = Util.findIndexByValue(this, event.target.innerText);
+    const index = Util.findIndexByValue(this, event.target.textContent);
```

Comment:
why `innerText` is replaced with textContent for focus issue !!

Reply:
can you share some details of the navigation issue ??  because textContent and innerText both are used for different scenarios so I think changing them might have some unintended effects on existing behavior.

Reply:
In Safari the index is not returning correctly if we use 'event.target.innerText', it is returning the index values as -1, -1, -1, 3 for 4 list items instead of 0, 1, 2, 3. when I change the index from 'event.target.innerText' to 'event.target.textContent' then the index values are returned as expected.